released/packages/coq-elm-extraction/coq-elm-extraction.0.1.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Coq extraction to Elm"
+description: """
+A framework for extracting Coq programs to Elm
+"""
+maintainer: "Danil Annenkov <danil.v.annenkov@gmail.com>"
+authors: "The COBRA team"
+license: "MIT"
+homepage: "https://github.com/AU-COBRA/coq-elm-extraction"
+dev-repo: "git+https://github.com/AU-COBRA/coq-elm-extraction.git"
+bug-reports: "https://github.com/AU-COBRA/coq-elm-extraction/issues"
+doc: "https://au-cobra.github.io/coq-elm-extraction/toc.html"
+
+depends: [
+  "coq" {>= "8.17" & < "9.1~"}
+  "coq-metacoq-utils" {>= "1.3.1" & < "1.4~"}
+  "coq-metacoq-common" {>= "1.3.1" & < "1.4~"}
+  "coq-metacoq-template" {>= "1.3.1" & < "1.4~"}
+  "coq-metacoq-template-pcuic" {>= "1.3.1" & < "1.4~"}
+  "coq-metacoq-pcuic" {>= "1.3.1" & < "1.4~"}
+  "coq-metacoq-safechecker" {>= "1.3.1" & < "1.4~"}
+  "coq-metacoq-erasure" {>= "1.3.1" & < "1.4~"}
+]
+
+build: [
+  [make "theory"]
+  [make "tests"] {with-test}
+]
+install: [
+  [make "install"]
+  [make "-C" "tests" "install"] {with-test}
```

Comment:
@4ever2 do you really want to install different files depending on whether opam was run with the test option or not? This means people who rely on this package can't be sure of what files are actually available.